.travis/after_success_lint.sh

```diff
@@ -3,14 +3,14 @@
 
 if [ "$TRAVIS_PULL_REQUEST" != 'false' ]; then
 	echo "Linting JavaScript files"
-	(lerna exec --scope=@talend/react-sagas -- npm run lint:es -- -o output/sagas.eslint.txt --no-color)
-	(lerna exec --scope=@talend/react-datagrid -- npm run lint:es -- -o output/datagrid.eslint.txt --no-color)
-	(lerna exec --scope=@talend/react-stepper -- npm run lint:es -- -o output/stepper.eslint.txt --no-color)
-	(lerna exec --scope=@talend/react-cmf -- npm run lint:es -- -o output/cmf.eslint.txt --no-color)
-	(lerna exec --scope=@talend/react-cmf-webpack-plugin -- npm run lint:es -- -o output/cmf-webpack-plugin.eslint.txt --no-color)
-	(lerna exec --scope=@talend/react-components -- npm run lint:es -- -o output/components.eslint.txt --no-color)
-	(lerna exec --scope=@talend/react-containers -- npm run lint:es -- -o output/containers.eslint.txt --no-color)
-	(lerna exec --scope=@talend/react-forms -- npm run lint:es -- -o output/forms.eslint.txt --no-color)
+	(lerna exec --scope=@talend/react-sagas -- npm run lint:es -- -o ../../output/sagas.eslint.txt --no-color)
```

Comment:
It uses @talend/scripts under the hood, and the cwd is not the root. It seems to be the case before, I don't understand why.